src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -224,6 +248,16 @@ private void clearCommand() {
 
     public void updateCurrentCommandField() {
         currentCommandField.setText(commandManager.getCurrentCommandString());
+        updateCommandHistoryField();  // Dodanie aktualizacji historii
+    }
+
+    public void updateCommandHistoryField() {
+        List<DriverCommand> history = commandManager.getCommandHistory();
+        StringBuilder historyText = new StringBuilder();
+        for (DriverCommand command : history) {
+            historyText.append(command.toString()).append("\n");
+        }
+        commandHistoryField.setText(historyText.toString());
```

Comment:
history should be more than a log. By clicking we should load the selected command to commandManager (setCurrentCommand).